src/common/RunParams.hpp

```diff
@@ -313,6 +317,10 @@ class RunParams {
   Index_type ltimes_num_g; /*!< num_g used in ltimes kernels (input option) */
   Index_type ltimes_num_m; /*!< num_m used in ltimes kernels (input option) */
 
+  Index_type femsweep_num_e; /*!< num_e used in femsweep kernel, number of elements (input option) */
```

Comment:
The problem size is fixed right? So this is not actually an input option?

Reply:
Yes, the problem size is fixed, but I thought to prepare it for mesh generation capability. This can be taken out though to prevent users from trying to set sizes.